src/System.Security.Cryptography.Xml/tests/XmlDsigXsltTransformTest.cs

```diff
@@ -182,8 +182,11 @@ public void OnlyInner()
             doc.LoadXml(test);
 
             transform.LoadInnerXml(doc.ChildNodes);
-            Stream s = (Stream)transform.GetOutput();
-            Assert.Throws<ArgumentNullException>(() => Stream2Array(s));
+            Assert.Throws<ArgumentNullException>(() =>
```

Comment:
@bartonjs we are porting existing Mono tests right now and simply fixing errors happened during conversion from different test framework. I agree this is not ideal but I think we should do separate pass on these kind of clean ups